src/test/java/me/karakelley/http/handlers/StaticFilesHandlerTest.java

```diff
@@ -1,59 +1,152 @@
 package me.karakelley.http.handlers;
 
-import me.karakelley.http.*;
-import me.karakelley.http.contentpresenter.ContentPresenter;
+import me.karakelley.http.HttpMethod;
+import me.karakelley.http.Request;
+import me.karakelley.http.Response;
 import me.karakelley.http.contentpresenter.HtmlPresenter;
 import me.karakelley.http.filesystem.PublicDirectory;
 import me.karakelley.http.helpers.TempFilesHelper;
-import me.karakelley.http.HttpMethod;
+import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 
 import java.nio.file.Path;
+import java.util.HashMap;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 class StaticFilesHandlerTest {
 
-  @Test
-  void testGetDisplayFiles() {
-    TempFilesHelper.withTempDirectory(directory -> {
-      Path fileOne = TempFilesHelper.createTempFile(directory, "/test1");
-      Path fileTwo = TempFilesHelper.createTempFile(directory, "/test2");
-      PublicDirectory publicDirectory = PublicDirectory.create(directory.toString());
 
-      ContentPresenter contentPresenter = new HtmlPresenter();
-      Handler handler = new StaticFilesHandler(publicDirectory, contentPresenter);
-      Response response = handler.respond(new Request(HttpMethod.GET, "/", "HTTP/1.1", null, null,0));
 
-      assertTrue(new String(response.getBody()).split("", 2).length == 2);
-    });
+  @Nested
+  class WhenRequestIsGet {
+    @Test
+    void testDisplaysFiles() {
```

Comment:
Is there a test for when the file isn't there?

Reply:
There is a test for this scenario in `HttpServerTest`